benchmarks/src/main/java/org/corfudb/benchmarks/SimpleTrace.java

```diff
@@ -0,0 +1,54 @@
+package org.corfudb.benchmarks;
+
+import lombok.extern.slf4j.Slf4j;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+public class SimpleTrace {
+    String name; //name of the tracing object
+    long total; //sum of latencise for all operations in microseconds
+    long cnt;   //total number of the operations.
+    long start; //the current operation start time in nanoseconds
+
+    SimpleTrace(String name) {
+        name = name;
```

Comment:
The scope of name field is shaded by the parameter. To assign that field, use "this.name = name"